0_reporting_functions.R

```diff
@@ -345,6 +344,14 @@ ReportGeneration <- function(){
     text <- removetextlines("OtherSectors")
   }
   
+  if(!IncCoalRetirement){
+    text <- removetextlines("CoalCapChart")
+  }
+  
+  if(!IncRenewableAddition){
+    text <- removetextlines("RenewableCapChart")
+  }
+  
```

Comment:
@jdhoffa @Clare2D shopuld we care about these being added to ReportGeneration()? My intuition says no, since we're about to get rid of it anyways...?

Reply:
Not sure I get the question - but this just removes the section for the "Renewable Additions chart" that you mention below as well. This can be removed if you're printing the chart, or if you remove this section manually from the tex file